src/app/pages/news/11.2/04-engine-cockpit.md

```diff
@@ -1,6 +1,13 @@
 ## Engine Cockpit {#engineCockpit}
 
-Blah Engine Cockpit blah
+We added awesome new world-class monitoring, user management, operations, and configuration features to the Engine Cockpit.
```

Comment:
sound like someone with pre-sales skills is talking :sweat_smile:   ... but don't get me wrong, I like it :D